streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/ThreadMetrics.java

```diff
@@ -83,6 +86,14 @@ private ThreadMetrics() {}
         "The average commit latency over all tasks assigned to one stream thread";
     private static final String COMMIT_OVER_TASKS_MAX_LATENCY_DESCRIPTION =
         "The maximum commit latency over all tasks assigned to one stream thread";
+    private static final String PROCESS_RATIO_DESCRIPTION =
+        "The fraction of time stream thread is spending on processing active tasks.";
```

Comment:
```suggestion
        "The fraction of time the thread spent on processing active tasks.";
```
(also applies to the others)

Reply:
Ack